package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "@arcadia-eng/utility-connect-react",
   "version": "0.0.1-alpha.9",
+  "engines": {
+    "npm": ">=7.0.0"
```

Comment:
@dlopuch let's stick with `npm` v6 for now. We can evaluate bumping to v7 in a... calmer week.